src/main/java/ua/zhdanova/hw20/dao/HibernatePersonDao.java

```diff
@@ -0,0 +1,80 @@
+package ua.zhdanova.hw20.dao;
+
+import org.hibernate.Session;
+import org.hibernate.Transaction;
+import ua.zhdanova.hw20.exception.DataProcessingException;
+import ua.zhdanova.hw20.model.Person;
+import ua.zhdanova.hw20.util.HibernateUtil;
+import java.util.List;
+
+public class HibernatePersonDao implements Dao<Person> {
+    @Override
+    public Person getById(Long id) {
+        try(Session session = HibernateUtil.getSessionFactory().openSession()) {
+            return session.get(Person.class, id);
+        } catch (Exception e) {
+            throw new DataProcessingException("Can't getting Person by ID: " + id, e);
+        }
+    }
+
+    @Override
+    public List<Person> getAll() {
+        try(Session session = HibernateUtil.getSessionFactory().openSession()) {
+            return session.createQuery("FROM Person", Person.class).list();
+        } catch (Exception e) {
+            throw new DataProcessingException("Can't get all Person's: ", e);
+        }
+    }
+
+    @Override
+    public Person add(Person entity) {
+        Transaction transaction = null;
+        Session session = null;
+        try {
+            session = HibernateUtil.getSessionFactory().openSession();
```

Comment:
Why not use try-with-resources here and in other places where we can use it.
Closing the resources is important.